docs/getting-started/development-environment.rst

```diff
@@ -178,7 +178,105 @@ images are having problems building.
 Linux
 ~~~~~
 
-Want to write it? Contact us on the chat!
+Useful packages to install:
+
+.. code-block:: sh
+
+        git:      $ apt install git-all
+
+        zoom:     https://support.zoom.us/hc/en-us/articles/204206269-Installing-or-updating-Zoom-on-Linux#h_adcc0b66-b2f4-468b-bc7a-12c182f354b7
+
+        discord:  https://doc.ubuntu-fr.org/discord
+        
+        VS code:  https://code.visualstudio.com/download
```

Comment:
There is already an editors section: https://invenio.readthedocs.io/en/latest/getting-started/development-environment.html#editors